packages/Pagination/src/components/Divider/Divider.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
```

Comment:
you can remove Divider.styles.js and move all into `Divider.js` Right now Divider.js is not doing anything apart of re-exporting `sc.Divider`, after moving everything you can simple delete Divider.styles.js and everything should work correctly. 

you can address the same as:

like:

```js
// Divider.js

import styled from "styled-components";
import tokens from "@paprika/tokens";

const Divider = styled.span`
  background-color: ${tokens.border.color};
  display: inline-block;
  height: 24px;
  margin-left: ${tokens.space};
  margin-right: ${tokens.space};
  width: 1px;
`;

export default Divider;
```